src/ensembl/src/store.ts

```diff
@@ -1,9 +1,48 @@
 import { applyMiddleware, createStore } from 'redux';
 import thunk from 'redux-thunk';
 import { composeWithDevTools } from 'redux-devtools-extension';
-
+import { createBrowserHistory } from 'history';
+import { routerMiddleware } from 'connected-react-router';
 import reducers from './rootReducer';
 
+import GoogleAnalyticsTracking from './services/AnalyticsService';
```

Comment:
We didn't discuss this, but I think the convention is that PascalCase, if it's used at all, is used only for naming of React component files. A service wouldn't be a component file, so I would advice the name `analytics-service`. @imran-ebi, wonder what you think.

Reply:
Yes, that would make it easier to differentiate the service files from the React components.

Reply:
Cool, I've renamed it.